CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
   This allows to have multiple json elements with the same name in
   different source files
 
+* The `lobster-codebeamer` tool now uses codebeamer api v3
```

Comment:
Please add the following additional comment:
```
Please note that the api v3 returns the value "Unset" for a codebeamer item status if the status is actually empty. The api v1 did not return any value at all for an item with a missing status. This means that the resulting lobster file will now contain "Unset" as status information, too, instead of `Null`.
```